Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs

```diff
@@ -31,9 +32,17 @@ public CandidateProfile()
             CreateMap<LocationDTO, Location>().ReverseMap();
 
             CreateMap<Candidate, ScheduleCandidateInfoDTO>()
-                .ForMember(dest => dest.ProjectResult, opt => opt.Ignore());
+                .ForMember(dest => dest.ProjectResult, opt => opt.Ignore())
+                .ForMember(dest => dest.Skype, conf => conf.MapFrom(src => src.Contacts.FirstOrDefault(c => c.Type == DataAccess.Constants.Contacts.Skype).Value));
+            CreateMap<CandidateDTO, ScheduleCandidateInfoDTO>()
+               .ForMember(dest => dest.ProjectResult, opt => opt.Ignore())
+               .ForMember(dest => dest.Skype, conf => conf.MapFrom(src => src.Contacts.FirstOrDefault(c => c.Type == DataAccess.Constants.Contacts.Skype).Value));
 
             CreateMap<CandidatesProjectInfo, CandidatesProjectInfoDTO>();
+            CreateMap<InterviewAppointmentDTO, CandidateRenewal>()
+                .ForMember(dest => dest.CandidateId, conf => conf.MapFrom(src => src.CandidateId))
+                .ForMember(dest => dest.IsAssignedOnInterview, conf => conf.MapFrom(src => src.IsAppointment))
+                .ForMember(dest => dest.Status, conf => conf.MapFrom(src => src.IsAppointment ? StatusDTO.TechInterviewOneStep : StatusDTO.RecruiterInterview));
```

Comment:
Where is tech 2 interview in your scheme? And what if it is appointment to recruiter interview? Status will be tech interview because flag is true?